src/constants.js

```diff
@@ -31,6 +31,7 @@ export const METADATA_PROVIDER_API = 'metadata-provider';
 export const MODES_OF_ISSUANCE_API = 'modes-of-issuance';
 export const NATURE_OF_CONTENT_TERMS_API = 'nature-of-content-terms';
 export const PATRON_GROUPS_API = 'groups';
+export const PERMISSIONS_USERS_API = 'perms/users';
```

Comment:
```suggestion
export const PERMISSION_USERS_API = 'perms/users';
```

Reply:
It is easier to pronounce and matches other constant names as well

Reply:
updated